packages/frames.js/src/middleware/openframes.ts

```diff
@@ -117,8 +117,11 @@ export function openframes<T = OpenFrameMessage>({
 
     const message = await handler.getFrameMessage(json);
```

Comment:
what happens if the message parsing fails here? Shouldn't we also handle that similarly to how it is handled in `farcaster` middleware?

Reply:
there is an explicit check if the openframes handler can handle a given payload before decoding, so i think it should fail if there is an error here